flow/project.py

```diff
@@ -836,24 +1010,25 @@ def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
         :rtype:
             bool
         """
-        return any(op.eligible(job, ignore_conditions) for op in self)
+        return any(op.eligible(jobs, ignore_conditions) for op in self)
 
-    def complete(self, job):
+    def complete(self, jobs):
         """True when all BaseFlowOperation post-conditions are met.
 
-        :param job:
-            A :class:`signac.Job` from the signac workspace.
-        :type job:
-            :class:`signac.Job`
+        :param jobs:
+            A list of :class:`signac.Job` from the signac workspace.
+        :type jobs:
+            list
         :return:
             Whether the group is complete (all contained operations are
             complete).
         :rtype:
             bool
         """
-        return all(op.complete(job) for op in self)
+        return all(op.complete(jobs) for op in self)
 
-    def add_operation(self, name, operation, directives=None):
+    def add_operation(self, name, operation, directives=None,
+                      flow_aggregate=None, flow_select=None):
```

Comment:
Users can no longer select `flow_aggregate` or `flow_select` for individual operations, correct? If that is the case it should be removed.